### ISSUE TITLE
UIAlertView Deprecated, use UIAlertController under iOS 8 with extra fields

### ISSUE THREAD

**Issue by lightbow** [mailchimp/ChimpKit3#5]
The current ChimpKit uses UIAlertView, which is great for iOS 7, but iOS 8 introduced UIAlertController. ChimpKit should check for iOS 8 and use the new class, which also provides the ability to add more than one text field. It would be great opportunity to have the user supply FirstName / LastName in addition to email address.

**Comment by parrots** [comment]
I'm a little worried to see no movement on this. UIAlertView is now deprecated, and I'm assuming iOS 10 may remove it completely.

**Comment by lightbow** [comment]
Right, anyone building with a target of iOS 9 (within the next 6 months or so, pretty much anyone who actually keeps their app up to date) or later will receive a compiler error:

`CKSubscribeAlertView.h:11:35: 'UIAlertView' is deprecated: first deprecated in iOS 9.0 - UIAlertView is deprecated. Use UIAlertController with a preferredStyle of UIAlertControllerStyleAlert instead`
